packages/genui/a2ui-playground/src/styles.css

```
@@ -324,18 +324,219 @@ a {
+  align-items: center;
+  gap: 5px;
+  color: inherit;
+}
+
+.phoneSignal {
+  width: 11px;
+  height: 9px;
+  opacity: 0.8;
+  background: linear-gradient(
+    to top,
+    currentColor 0 18%,
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)